README.md

```diff
@@ -47,15 +47,23 @@ will generated markup like:
 
 ### With Preprocessors
 
-To use this addon with another preprocessor (such as Sass or Less), simply import `pod-styles` into your base app stylesheet.
-
-For example, if you're using Sass:
+To use this addon with a CSS pre-processor, import `pod-styles` into your base stylesheet. (see your pre-processor docs)
 
 ```scss
 // app/styles/app.scss
 @import "pod-styles";
 ```
 
+```less
+// app/styles/app.less
+@import "pod-styles.less";
```

Comment:
if the only are only working with less files (or any other file types), then you don't have to add the extension. It will create a pod-styles.EXT for every component style file EXT that you have. So you only need to add the EXT when you are dealing with conflicts arising from having multiple different types of style files defined.

So.. maybe add in something to the effect of `if you are using more then one type of component style files (ie a .less file and a .scss file) then you will need to add the extension. Otherwise the extension can be left off.` Does that sound good and make sense?